frontend/packages/dev-console/src/components/monitoring/overview/MonitoringOverviewAlerts.tsx

```diff
@@ -0,0 +1,49 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { Alert } from '@patternfly/react-core';
+import { Link } from 'react-router-dom';
+import { fromNow } from '@console/internal/components/utils/datetime';
+import { Alert as AlertType } from '@console/internal/components/monitoring/types';
+import { labelsToParams } from '@console/internal/components/monitoring/utils';
+import { sortMonitoringAlerts } from '@console/shared';
+import { getAlertType } from './monitoring-overview-alerts-utils';
+import './MonitoringOverviewAlerts.scss';
+
+interface MonitoringOverviewAlertsProps {
+  alerts: AlertType[];
+}
+
+const MonitoringOverviewAlerts: React.FC<MonitoringOverviewAlertsProps> = ({ alerts }) => {
+  const sortedAlerts = sortMonitoringAlerts(alerts);
+
+  return (
+    <div className="odc-monitoring-overview-alerts">
+      {_.map(sortedAlerts, (alert: AlertType) => {
+        const {
+          activeAt,
+          annotations: { message },
+          labels: { severity, alertname, namespace },
+          rule: { name, id },
+        } = alert;
+        const alertDetailsPageLink = `/dev-monitoring/ns/${namespace}/alerts/${id}?${labelsToParams(
```

Comment:
`dev-monitoring` should be not be hardcoded here. It should be based on an active perspective. Because overview panel has been used in both the perspective.

Reply:
The metrics section in the side panel has a similar behavior. can we fix this as a bug?

Reply:
sure, can take this as issue. Pls log an issue for it

Reply:
Bug: https://issues.redhat.com/browse/ODC-4397